### ISSUE TITLE
python 2.7

### ISSUE THREAD

**Issue by vdomos** [theolind/pymysensors#83]
Hi,

This API seem only work with Python 3, 
I have a error when inporting the module "queue". Not found in Debian packages or pip.

Do you know how to make it work in python 2.7 ?
It is to integrate it in a home automation software under Python 2.7.

**Comment by MartinHjelmare** [comment]
Yes, the API is written for python 3. I'm not sure it's easily done to support python 2.7. I can have a look. What home automation software are you planning to use?

**Comment by vdomos** [comment]
It's Domogik http://domogik.org/en/, a small for now home automation software written in Python 2.7.
There is the possibility of adding plugins to control different devices like zwave, onewire, rfxcom ...
I'm not the developper of this software, only for simple plugins.

I recently discovered MySensors, test some nodes.
I found your API interesting to help me use MySensors with Domogik  but I must first work on the MySensor 's documentation to fully understand how it works.

I do not want to give you more work
